.gitignore

```diff
@@ -22,7 +22,6 @@ storybook-static
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-.vscode
```

Comment:
should settings need to be included or we can just use a different approach as I was suggesting here?

https://github.com/informatici/openhospital-ui/pull/645#issuecomment-2277401833

Reply:
I think we can include `.vscode` file to let vscode users share useful configs. `.ide-settings/` is generic and is not automatically processed by vscode, I think.

Reply:
is not possible to add a `vscode` folder in `.ide-settings/` like we have done for other IDEs?

Reply:
Yeah, we can do that. 
So @gasp if you agree, create the folder `.ide-settings`, add .vscode in that folder. So that you can let .vscode in the root folder excluded.

Reply:
so we can close this PR?

Reply:
If @gasp agree, yes